### PR TITLE
feat: added `floor_mod` in paddle frontend

### LINKED ISSUE #26059: floor_mod

### PR DESCRIPTION
**Pull request by AbdullahSabry**
Close #26059

### CODE REVIEW

File: ivy/functional/frontends/paddle/tensor/tensor.py [hunk 1]
```diff
@@ -606,6 +606,12 @@ def isclose(self, y, rtol=1e-05, atol=1e-08, equal_nan=False, name=None):
     def floor_divide(self, y, name=None):
         return paddle_frontend.floor_divide(self, y)
 
+    @with_supported_dtypes(
+        {"2.5.1 and below": ("float32", "float64", "int32", "int64")}, "paddle"
+    )
+    def floor_mod(self, y, name=None):
+        return paddle_frontend.Tensor(ivy.remainder(self._ivy_array, _to_ivy_array(y)))
```
Comment (abdulasiraj) [inline ivy/functional/frontends/paddle/tensor/tensor.py:613]: Hey @AbdullahSabry 
no need to use casting methods as array is already `ivy`. so, you can use `paddle_frontend.remainder(self, y)` directly.
thanks!

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py [hunk 2]
```diff
@@ -1995,6 +1995,45 @@ def test_paddle_tensor_floor_divide(
...
+    init_tree="paddle.to_tensor",
+    method_name="floor_mod",
+    dtypes_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+        num_arrays=2,
+        min_value=2,
+        shared_dtype=True,
+        large_abs_safety_factor=2,
+        small_abs_safety_factor=2,
+        safety_factor_scale="linear",
```
Comment (abdulasiraj) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py:2010]: I think we can also omit safety_factor_scale here

**Review by abdulasiraj — CHANGES_REQUESTED** [review]

File: ivy/functional/frontends/paddle/tensor/tensor.py (hunk 1 above)
Comment (AbdullahSabry) [reply ivy/functional/frontends/paddle/tensor/tensor.py]: @abdulasiraj 
Yes you're right

**Comment by AbdullahSabry** [comment]
@abdulasiraj 
Pushed commit with requested changes!

**Comment by NripeshN** [comment]
LGTM!!

**Pull request merged**